profiles/common/modules/features/ec_embedded_image/ec_embedded_image.behat.inc

```diff
@@ -86,34 +86,19 @@ class ECEmbeddedImageSubContext extends DrupalSubContextBase {
       ]
     );
 
-    $this->contentTypeWasSetUp = TRUE;
+    $this->permissionWasSetUp = TRUE;
   }
 
   /**
    * Removes the test_media_gallery content type and related permissions.
    *
    * @AfterScenario
    */
-  public function removeMediaGalleryContentType() {
-    if (!$this->contentTypeWasSetUp) {
+  public function resetMediaGalleryPermission() {
+    if (!$this->permissionWasSetUp) {
       return;
     }
 
-    $this->contentTypeWasSetUp = FALSE;
```

Comment:
I was a mistake but after reading this doc: Your context class is initialized before each scenario is run, and every scenario has its very own context instance. It appears that this line is useless.

So, I will keep it like that.